rcutils/logging.py

```diff
@@ -54,7 +54,8 @@
 skipfirst_doc_lines = [
     'The first log call is being ignored but all subsequent calls are being processed.']
 throttle_params = OrderedDict((
-    ('time_source_type', 'The time source type of the time to be used'),
+    ('time_source_type', 'Function that returns rcutils_ret_t and expects a '
+        'rcutils_time_point_value_t pointer.'),
```

Comment:
@BMarchi nit: I wonder if `time_source` makes more sense than `time_source_type`. @dirk-thomas thoughts?

Reply:
Well, "time source" has a very specific meaning in `rclcpp` and since this is not the same I would avoid using the same term for it. See my other comment for a suggestion.

Reply:
So I should call it then `get_time_point_value`?